lib/job.js

```diff
@@ -229,6 +229,12 @@ class Job extends BaseModel {
                     apiUri: this[apiUri],
                     isUpdate: true
                 });
+            } else {
+                await this[executor].stopPeriodic({
```

Comment:
we will be making this Queue service call for each `job.update()` and this happens all the time as part of `pipeline.sync()`, 
1. when webhooks are fired, 2. when a build is started. 

While this is the right thing to do when `buildPeriodically` config is removed, I'm afraid this will lead to increase in load of queue service by many fold. 

So we should be doing this only if the config was there before. (like the comment I've left [here](https://github.com/screwdriver-cd/models/pull/465/files#diff-d823dceb04482ab55e5004eebb53fc1cR222))

Reply:
I fixed it.